scope.c

```diff
@@ -1348,7 +1348,12 @@ Perl_leave_scope(pTHX_ I32 base)
 	    break;
 
 	case SAVEt_HINTS:
+        {
```

Comment:
The perl style is to indent the braces to the same level as the `case` keyword. Aslo, new code should use spaces for identation, not tabs.

Reply:
Ah I was wondering about spaces vs. tabs; this file is full of tabs all around the area I'm working. But I can adjust my newly-added lines to be spaces.

Reply:
Also, if you view the file outside of github you'll see the braces *are* aligned with the `case`. The difference is that the `case` line begins with a tab, whereas the braces line starts with 8 spaces. Because of the two leading columns of `diff` output to print the "+ " here, it means that visually the brace looks like it's in column 10, while the `case` is still on column 8.

Reply:
See also - https://twitter.com/cpan_pevans/status/1277026253711777793

But in summary, what should I do here? My newly-added brace line is space-indented, the previous case line is tabbed. They do align for real, their alignment is just being messed up by diff. Should I leave the code as it is, or edit the `case` line to be space-indented too?

Reply:
my $0.02: make the minimum number of changes in this commit, and then we can reformat the file later in a different commit/PR to make everything consistent.

Reply:
OK; as of 1a15b8e0a5 all the newly-added lines are done with only spaces. That things don't appear to line up is a github/diff artefact of mixed spaces/tabs - see my tweet link. I am happy to assert it does line up nicely when viewed for real.